editor/animation_track_editor.cpp

```diff
@@ -2699,7 +2699,11 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 						StringName prop = npath.get_concatenated_subnames();
 						PropertyInfo prop_info;
 						ClassDB::get_property_info(nd->get_class(), prop, &prop_info);
+#ifdef DISABLE_DEPRECATED
+						bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians_as_degrees") != -1;
+#else
 						bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians") != -1;
+#endif // DISABLE_DEPRECATED
```

Comment:
You need to test both in the case where `DISABLE_DEPRECATED` is not defined. Supporting deprecated syntax doesn't mean not supporting the new syntax, it means supporting both (new first, and fallback to deprecated).

Reply:
Well `prop_info.hint_string.find("radians") != -1` implies `prop_info.hint_string.find("radians_as_degrees") != -1`

Reply:
Ah indeed!

Reply:
Granted it will allow anything including `radians` but I'm not sure how to test cleanly here for whole words, and the full phrase would also accept things like `aaaaradians_as_degreesaaaa`